packages/_example-extractor/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@krassowski/jupyterlab-lsp-example-extractor",
+  "description": "Example extractor package from @krassowski/jupyterlab-lsp",
+  "version": "0.0.0",
+  "private": true,
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "author": "JupyterLab-LSP Development Team",
+  "files": [
+    "lib/**/*.{js,ts}"
+  ],
+  "dependencies": {
+    "@krassowski/jupyterlab-lsp": "^3.4"
```

```suggestion
    "@krassowski/jupyterlab-lsp": "^3.5"
```
Though the versions need to be bumped elsewhere for it to compile I think

---

yeah, seems out of scope for this pr...

---

sure, can do it later, just noted to remember!